environment.yml

```diff
@@ -28,6 +28,7 @@ dependencies:
   - flask-cors==4.0.0
   - redis-py==5.0.1
   - botocore>=1.33.10 # Minimum version that is compatible with python >= 3.10 is botocore>=1.13.0
+  - h5netcdf
```

Comment:
It is required for loading netcdf file into memory from S3 bucket, used in `s3_connection.py` line 104:
`xr.load_dataset(body_object, engine="h5netcdf")`

Reply:
That sounds good. Could you lock down the version for it as well, for consistent builds.

I looked into the library and it looks like it can be a drop-in replacement for `python-netcdf4` which I thought we used but looking closer we don't use it anymore.

Reply:
locked down h5netcdf version to 1.3.0